spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jasmine */
+/* global GOVUK */
+
+describe('The intervention component', function () {
+  const FIXTURE = `
+    <section class="gem-c-intervention" role="region" data-module="intervention">
+      <h2>
+        <a href="/next-steps-for-your-business">
+          Check the next steps for your limited company
+        </a>
+      </h2>
+
+      <p class="govuk-body"></p>
+      <p class="govuk-body"></p>
```

Comment:
I really don't think you need any markup in this component fixture at all - it won't make any difference to the outcome of the test.